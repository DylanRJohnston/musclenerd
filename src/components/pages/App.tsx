import React from "react"

import { LandingPage } from "../layouts/LandingPage"

export const App: React.FC = () => <LandingPage text="Hello World" />
