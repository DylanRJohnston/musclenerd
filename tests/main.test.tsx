import React from "react"
import renderer from "react-test-renderer"

import { App } from "../src/components/pages/App"

test("Main loads without error", () => {
  const component = renderer.create(<App />)

  expect(component.toJSON()).toMatchSnapshot()
})
