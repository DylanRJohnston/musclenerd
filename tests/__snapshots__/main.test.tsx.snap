// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Main loads without error 1`] = `
<div>
  <div>
    <h1>
      Hello World
    </h1>
  </div>
</div>
`;
